### PR TITLE
Jeanette development2

### PR DESCRIPTION
**Pull request by jayyduran0108**
I added a biography option for the profile pages. I left my test function commented out because it was not working for my prototype (I know this might not be good but it keeps failing when I add it). If you can see something wrong with it that would be helpful thanks!

### CODE REVIEW

File: src/main/java/codeu/controller/ProfileServlet.java [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import org.jsoup.Jsoup;
+import org.jsoup.safety.Whitelist;
+
+public class ProfileServlet extends HttpServlet {
+
+    /** Store class that gives access to Users. */
```
Comment (chilimalone) [inline src/main/java/codeu/controller/ProfileServlet.java]: sorry for constant comments on this - I keep seeing either +4 indent or +2 indent, and I'm no longer sure which one I advised where. Let's go with +2 spaces for indent from here on out

File: src/main/java/codeu/controller/ProfileServlet.java [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+import java.util.UUID;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import org.jsoup.Jsoup;
+import org.jsoup.safety.Whitelist;
+
+public class ProfileServlet extends HttpServlet {
```
Comment (chilimalone) [inline src/main/java/codeu/controller/ProfileServlet.java]: Add class comments? perhaps something like "Serves profile pages"?

File: src/main/java/codeu/controller/ProfileServlet.java [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+
+    /** Store class that gives access to Users. */
+    private UserStore userStore;
+
```
Comment (chilimalone) [inline src/main/java/codeu/controller/ProfileServlet.java]: let's go with only 1 newline between methods/fields (applies for rest of file)

File: src/main/java/codeu/controller/ProfileServlet.java [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+    }
+
+  
+    /**
+    * This function fires when a user navigates to the profile page.
+    */
+    @Override
+    public void doGet(HttpServletRequest request, HttpServletResponse response)
+        throws IOException, ServletException {
+
```
Comment (chilimalone) [inline src/main/java/codeu/controller/ProfileServlet.java]: nit: remove extraneous new lines?

File: src/main/webapp/WEB-INF/view/profile.jsp [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+<head>
+  <title> Profile </title>
+  <link rel="stylesheet" href="/css/main.css" type="text/css">
+</head>
+<body>
+  <nav>
+    <a id="navTitle" href="/">CodeU Chat App</a>
+    <a href="/conversations">Conversations</a>
+      <% if (request.getSession().getAttribute("user") != null) { %>
+    <a> Hello <%= request.getSession().getAttribute("user") %>!</a>
```
Comment (chilimalone) [inline src/main/webapp/WEB-INF/view/profile.jsp]: +2 indent for being inside <% if

File: src/test/java/codeu/controller/ProfileServletTest.java [hunk 6]
```diff
@@ -0,0 +1,59 @@
+/*package codeu.controller;
```
Comment (chilimalone) [inline src/test/java/codeu/controller/ProfileServletTest.java]: ? remove commented out code/classes

File: src/main/java/codeu/controller/ProfileServlet.java [hunk 7]
```diff
@@ -0,0 +1,103 @@
...
+import codeu.model.data.Message;
+import codeu.model.data.User;
+import codeu.model.store.basic.ConversationStore;
+import codeu.model.store.basic.MessageStore;
+import codeu.model.store.basic.UserStore;
+
+import java.io.IOException;
+import java.time.Instant;
+import java.util.List;
+import java.util.UUID;
```
Comment (Georgeg10499) [inline src/main/java/codeu/controller/ProfileServlet.java]: Maybe leave a space in between line 13 and 14

**Review by Georgeg10499 — APPROVED** [review]

File: src/main/java/codeu/controller/ProfileServlet.java (hunk 2 above)
Comment (jayyduran0108) [reply src/main/java/codeu/controller/ProfileServlet.java]: done

File: src/main/java/codeu/controller/ProfileServlet.java (hunk 7 above)
Comment (jayyduran0108) [reply src/main/java/codeu/controller/ProfileServlet.java]: done

**Pull request merged**